### DISCUSSION TITLE
Keyboard shortcut for go to line

### DISCUSSION THREAD

**Discussion by melissachang** [community/community#9671] [Other Feature Feedback, Questions, & Ideas]
Often times I want to go to a specific line. For example, I'm looking at this test failure:

![image](https://user-images.githubusercontent.com/10929390/148266212-e39a3767-a3f0-450f-918c-75582c959634.png)

So I want to go to `BigQueryCowTest.java` line 123.

**Comment by look** [comment]
Thanks for the idea. This exists in the GitHub.com UX using the `l` key, so it would make sense for us to support it the same way. 👍

**Reply by 0xdevalias** [reply]
Does this still exist from the `l` key? It doesn't seem to work for me?

**Comment by colinwm** [comment]
We do support keyboard shortcuts for jumping to a line if you already have the file open. E.g. if you open this: https://cs.github.com/DataBiosphere/terra-cloud-resource-lib/blob/fdf26f8fd75325bc7d272864cd4a3122f6a102f9/google-bigquery/src/test/java/bio/terra/cloudres/google/bigquery/BigQueryCowTest.java?q=BigQueryCowTest#L1

and then type `123<Enter>`, you'll jump to that line.  You can also press `?` to see the list of all keyboard shortcuts.

**Comment by melissachang** [comment]
Neat, thanks. I did look at `?`, the shortcut you described isn't documented there.

FWIW I think `<some-letter>123` is more intuitive than `123<enter>`.

**Comment by 0xdevalias** [comment]
I would like to be able to use the `t` (Jump to file) shortcut, and give it a full path, optionally also with a line number, such as `foo/bar/baz/index.js:123` and have it jump to the correct file and line.

I believe it used to be possible to use this with the file path (without line number), but now even that seems to fail using the new code search/etc beta.

**Reply by 0xdevalias** [reply]
> I would also like to be able to paste a full file path, with optional line number into the cmd+k dialog, and have it also suggest opening that file, at that line number.
>
> _Originally posted by @0xdevalias in https://github.com/orgs/community/discussions/39163#discussioncomment-4385754_

**Reply by 0xdevalias** [reply]
See also: https://github.com/orgs/community/discussions/38692#discussioncomment-4100798

**Reply by colinwm** [reply]
This is a nice idea, and we should add support for it in the `t` shortcut and the new code search beta.
